app/assets/javascripts/app/views/help_view.js

```diff
@@ -187,8 +197,23 @@ app.views.Help = app.views.StaticContentView.extend({
     e.preventDefault();
   },
 
+  chat: function(e){
+    this.renderStaticSection("chat", "faq_chat", this.CHAT_SUBS);
+    this.menuClicked(e);
+
+    e.preventDefault();
+  },
+
   linkHtml: function(url, text) {
     return "<a href=\"" + url + "\" target=\"_blank\">" + text + "</a>";
+  },
+
+  linkImage: function(src, alt){
+    return '<div class="help-image"><img src="images/' + src + '" alt="' + alt + '"/></div>';
```

Comment:
Does `public/assets/` exist? Try removing it. Also try `bin/rake tmp:cache:clear` and restarting the app server.